app/views/templates/credit_note.slim

```diff
@@ -326,61 +326,61 @@ html
           .body-2 = customer.email
 
       .mb-24
-        h2.title-2.mb-8 = total_amount.format
+        h2.title-2.mb-8 = total_amount.format(format: I18n.t('money.format'), decimal_mark: I18n.t('money.decimal_mark'), thousands_separator: I18n.t('money.thousands_separator'))
         .body-1
           - if credited? && refunded?
-            | Credited on customer balance and refunded on #{issuing_date.strftime('%b %d, %Y')}
+            = I18n.t('credit_note.credited_refunded_notice', issuing_date: I18n.l(issuing_date, format: :default))
```

Comment:
Is it mandatory to put `format: :default` or used by default?

Reply:
yep in this case, since default is just key name in yml file